backend/gasless/src/main.ts

```diff
@@ -0,0 +1,144 @@
+import { GearApi, HexString, VoucherIssuedData, IUpdateVoucherParams } from '@gear-js/api';
+import { waitReady } from '@polkadot/wasm-crypto';
+import { hexToU8a, } from '@polkadot/util';
+
+import { Keyring } from '@polkadot/api';
+
+const secondsToBlock = 3;
+
+export class GaslessService  {
```

Comment:
Don't we want the server to work out of the box. It looks like there is no implementation of server controllers